packages/table/src/Table.ts

```diff
@@ -475,6 +461,17 @@ export class Table extends SizedMixin(SpectrumElement, {
                     );
                 }
             );
+            this.tableBody.addEventListener(
```

Comment:
Just for understanding - why is this event re-dispatched instead of bubbling?

Reply:
This "normalizes" to what Lit is doing in their element: https://github.com/lit/lit/blob/main/packages/labs/virtualizer/src/events.ts Previously, they dispatched it bubbling, but not in the new version.

By default, we create the `sp-table-body` where this event is actually dispatched, which makes listening on it precarious, so we mimic the event on the `sp-table` that the consuming dev does own to make it as much like consuming a `lit-virtualizer` directly.